docs/api/modules/ChainOfThought.md

```diff
@@ -64,3 +61,15 @@ pred = generate_answer(question=question)
 print(f"Question: {question}")
 print(f"Predicted Answer: {pred.answer}")
 ```
+
+The following example shows how to specify your custom rationale.
+
+```python
+#define a custom rationale
+rationale_type = dspy.OutputField(
+            prefix="Reasoning: Let's think step by step in order to",
+            desc="${produce the answer}. We ...",
```

Comment:
Lets clarify that it might not only be producing an answer. Could be any last key